test/protocol/clients/BosonVoucher.js

```diff
@@ -504,6 +504,30 @@ describe("IBosonVoucher", function () {
       });
     });
 
+    context("getRoyaltyPercentage()", function () {
+      it("should return the royalty fee percentage", async function () {
+        // give ownership to operator
+        await bosonVoucher.connect(protocol).transferOwnership(operator.address);
+
+        royaltyPercentage = "1000"; //15%
+        await bosonVoucher.connect(operator).setRoyaltyPercentage(royaltyPercentage);
+
+        expect(await bosonVoucher.connect(rando).getRoyaltyPercentage()).to.equal(
+          royaltyPercentage,
+          "Invalid royalty percentage"
+        );
+      });
+    });
+
+    context("getSellerId()", function () {
```

Comment:
You put the `getSellerId()` context inside `ERC2981 NFT Royalty fee` context, but it does not realy belog here. Just put it at the same level as `issueVoucher()`, `burnVoucher()` etc.